### PR TITLE
[ADD]: l10n_se: Add Sweden K3 menu

### PR DESCRIPTION
**Pull request by JulienAlardot**
[ADD]: l10n_se: Add K3 BS and PnL reports menu items, tags, and accounts

As there are currently no menu for the implementation
of Sweden localisation Balance sheet and Profit and Loss reports.
Links for those forms are added here.

Are also added a K2 complete COA  implementation and
a complete child COA that is BAS compliant.

Tags for more account controls on reports were also added.

Task: task-2674123

Description of the issue/feature this PR addresses: Sweden localization for accounting 

Current behavior before PR: No  Access to the Sweden Report Documents

Desired behavior after PR is merged: Access to the Sweden Report Documents

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_se/i18n/l10n_se.pot [hunk 1]
```diff
@@ -30,12 +30,12 @@ msgstr ""
 #. module: l10n_se
 #: model:ir.model,name:l10n_se.model_res_company
 msgid "Companies"
-msgstr ""
+msgstr "Företag"
```
Comment (jco-odoo) [inline addons/l10n_se/i18n/l10n_se.pot:33]: Translations are only in the .po file (pot -> template)

File: addons/l10n_se/data/menuitem_data.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem id="account_reports_se_statements_menu" name="Sweden K3" parent="account.menu_finance_reports" sequence="0" groups="account.group_account_readonly"/>
```
Comment (jco-odoo) [inline addons/l10n_se/data/menuitem_data.xml:3]: I would just call it Sweden to avoid confusion?

File: addons/l10n_se/__manifest__.py [hunk 3]
```diff
@@ -12,7 +12,9 @@
 This is the base module to manage the accounting chart for Sweden in Odoo.
 It also includes the invoice OCR payment reference handling.
     """,
-    "depends": ["account", "base_vat"],
+    "depends": ["account",
+                "l10n_multilang",
```
Comment (jco-odoo) [inline addons/l10n_se/__manifest__.py:16]: we should not change dependencies in stable and it is also not necessary here.

**Review by jco-odoo — CHANGES_REQUESTED** [review]

File: addons/l10n_se/i18n/l10n_se.pot [hunk 4]
```diff
@@ -192,3 +192,8 @@ msgid ""
 "You can choose different models for each type of reference. The default one "
 "is the Odoo reference."
 msgstr ""
+
+#. module: l10n_se
+#: model:ir.ui.menu,name:l10n_se.account_reports_se_statements_menu
+msgid "Sweden K3"
```
Comment (william-andre) [inline addons/l10n_se/i18n/l10n_se.pot:198]: This is not sorted alphabetically

You can have a look at https://man7.org/linux/man-pages/man1/msgcat.1.html

File: addons/l10n_se/data/account.account.template.csv [hunk 5]
```diff
@@ -1,325 +1,328 @@
...
+a1750,1750,Upplupna hyresintäkter,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1760,1760,Upplupna ränteintäkter,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1790,1790,Övriga förutbetalda kostnader och upplupna intäkter,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_5,False
+a1810,1810,Andel i börsnoterade företag,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_6,False
+a1880,1880,Andra kortfristiga placeringar,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_6,False
+a1890,1890,Nedskrivning av kortfristiga placeringar,account.data_account_type_current_assets,l10nse_chart_template,l10n_se.account_tag_6,False
+a1910,1910,Kassa,account.data_account_type_liquidity,l10nse_chart_template,l10n_se.account_tag_7,True
+a1920,1920,PlusGiro,account.data_account_type_liquidity,l10nse_chart_template,l10n_se.account_tag_7,True
+a1930,1930,Företagskonto/checkkonto/affärskonto,account.data_account_type_liquidity,l10nse_chart_template,l10n_se.account_tag_7,True
```
Comment (qdp-odoo) [inline addons/l10n_se/data/account.account.template.csv:57]: 1910, 1920, 1930: liquidity +allow reconciliation = dubious. IMO these accounts should be simply removed. @tsb-odoo what do you think?

**Review by qdp-odoo — CHANGES_REQUESTED** [review]
control_domain needs to be re-done. We can't create tags just for the control domains

File: addons/l10n_se/data/account_account_tag.xml [hunk 6]
```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <!-- Account Tags -->
+        <record id="account_tag_0" model="account.account.tag">
+            <field name="name">Subscribed but unpaid capital</field>
+            <field name="applicability">accounts</field>
```
[the remark this answers is not in the document]
Comment (JulienAlardot) [reply addons/l10n_se/data/account_account_tag.xml:7]: It didn't worked with account types as some mixed types are often present under some report lines

File: addons/l10n_se/data/account.account.template.csv (hunk 5 above)
Comment (JulienAlardot) [reply addons/l10n_se/data/account.account.template.csv:57]: Those accounts seems to be used a lot in the reports examples/samples I found, and those lines were not added by me (just the tags) but on the commit [a0f0cdd7](https://github.com/odoo-dev/odoo/commit/a0f0cdd70e88e9da322b9e02157c58e233ba7104)
I can change the reconciliation if needed

**Comment by qdp-odoo** [comment]
> control_domain needs to be re-done. We can't create tags just for the control domains

actually, after discussion with @tsb-odoo it's actually fine like that. Just make sure that the higher levels of the rapports use the type of accounts instead of relying on tags.

File: addons/l10n_se/data/account.account.template-K3.csv [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+a8320,8320,"Värdering till verkligt värde, omsättningstillgångar",account.data_account_type_other_income,l10nse_chart_template_K3,l10n_se.account_tag_33,False
+a8321,8321,Orealiserade värdeförändringar på omsättningstillgångar,account.data_account_type_other_income,l10nse_chart_template_K3,l10n_se.account_tag_33,False
+a8325,8325,Ränteintäkter från omsättningstillgångar,account.data_account_type_other_income,l10nse_chart_template_K3,l10n_se.account_tag_33,False
+a8450,8450,Orealiserade värdeförändringar på skulder,account.data_account_type_other_income,l10nse_chart_template_K3,l10n_se.account_tag_33,False
+a8451,8451,Orealiserade värdeförändringar på skulder,account.data_account_type_other_income,l10nse_chart_template_K3,l10n_se.account_tag_33,False
+a8455,8455,Orealiserade värdeförändringar på säkringsinstrument,account.data_account_type_other_income,l10nse_chart_template_K3,l10n_se.account_tag_33,False
+a8480,8480,Aktiverade ränteutgifter,account.data_account_type_other_income,l10nse_chart_template_K3,l10n_se.account_tag_33,False
+a8940,8940,Uppskjuten skatt,account.data_account_type_expenses,l10nse_chart_template_K3,l10n_se.account_tag_35,False
```
Comment (william-andre) [inline addons/l10n_se/data/account.account.template-K3.csv:27]: Check that your files end with a new line char

File: addons/l10n_se/data/account_account_tag.xml [hunk 8]
```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (william-andre) [inline addons/l10n_se/data/account_account_tag.xml:1]: Might be more readable/easier to manipulate in CSV

**Review by william-andre — COMMENTED** [review]
Just details:

It is missing the task id in the commit message, and the link to the commit in enterprise is error prone as you will most likely forget to update it when changing that commit.

The link will be added anyways by the mergebot

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**